woocommerce/woocommerce-ios
# [Blaze] Suggest CTA text using AI

Closes: #14376


## Description
This PR updates the Blaze suggestion API to include the `cta_text` field, and use it in the campaign creation flow.

## Steps to reproduce
1. Use a blaze eligible website.
2. Open the app.
3. Start blaze campaign creation.

## Testing information
- Confirm that the CTA text is pre-filled like tagline and description.
- Edit the ad and confirm you can select different versions of the CTA text.
- Make manual changes, and confirm they work without issues.

## Screenshots
https://github.com/user-attachments/assets/7c1936d2-b078-41a2-be86-c42f655f27fd

---

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
